src/layouts/sidebar-sidecar/server.ts

```diff
@@ -27,11 +27,13 @@ export function getStaticGenerationFunctions({
   basePath,
   baseName,
   additionalRemarkPlugins = [],
+  mainBranch,
```

Comment:
[hashicorp/packer](https://github.com/hashicorp/packer) uses `master` as the default branch, not `main`. I've exposed the `mainBranch` configuration option, as optional, here so we can configure the loader appropriately. The main user-facing impact here is that `Edit on GitHub` links, which rely on `mainBranch` being correct, work as expected with this change in place.